apps/src/applab/applab.js

```diff
@@ -830,6 +832,8 @@ Applab.reset = function(first) {
     applabTurtle.turtleSetVisibility(true);
   }
 
+  designMode.addKeyboardHandlers();
```

Comment:
We blow away divApplab on a semi-regular basis (and with it, any attached events).